Add instructions to README regarding benchmarking on pre ROCm 6.4.x v… (#114)
ROCm/rccl-tests: README.md

See the [Performance](doc/PERFORMANCE.md) page for explanation about numbers, and in particular the "busbw" column.

### Environment variables

> On some older versions of ROCm before 6.4.0, setting HSA_NO_SCRATCH_RECLAIM=1

Reviewer: ```suggestion
On some older versions of ROCm before 6.4.0, setting `HSA_NO_SCRATCH_RECLAIM=1`
```

>  as part of the environment may be necessary to achieve better performance.  When running without MPI, something like the following is enough:

Reviewer: ```suggestion
 as part of the environment might be necessary to achieve better performance.  When running without MPI, a command similar to the following one should be sufficient:
```

```shell
HSA_NO_SCRATCH_RECLAIM=1 ./build/all_reduce_perf -b 8 -e 128M -f 2 -g 8
```

> For MPI, you may need to do something like the following:

Reviewer: ```suggestion
For MPI, you might need to use a command similar to the following:
```
